### ISSUE TITLE
HW#1) Path problem ex)../qemu

### ISSUE THREAD

**Issue by jaeckiee** [csi2115-f21/scripts#9]
`git clone https://github.com/csi2115-f21/scripts`
`cd scripts`
`./install-deps.sh`
`git clone https://github.com/csi2115-f21/qemu`

If you excute these commands, the directory of qeme will be in the directory of scripts.
So, you need to excute `cd ..`, before excuting `git clone https://github.com/csi2115-f21/qemu`

**Comment by limsanky** [comment]
Yep, you need to be on the root directory and then `cd ...` into a new directory!

**Comment by aitnesearp** [comment]
Yeah,
But there are also cases where you shouldn't execute `cd ..`
like when running the CSI2115 Virtual Machine with `$ ./run-qemu.sh` .
In this case, you have to  `$ cd scripts` before running the code even though it isn't mentioned in the assignment!
